Header header
int64 length #Length of data 
string EPC   #EPC Data
unit32 port  # port number 
unit64 count # number  of reads
